tb_plugin/packaging/torch_tb_profiler/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: torch_tb_profiler
+  version: "{{ environ.get('BUILD_VERSION', '0.2.0') }}"
+
+source:
+ path: "{{ environ.get('SOURCE_ROOT_DIR', '../..') }}"
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pytorch
+
+  run:
+    - python
+    - pandas
+    - tensorboard
+
+build:
+  string: py{{py}}
```

Comment:
Since this package only contains python scripts, is it better to specify "noarch: python" for package compatible with all platforms?

Reply:
Good point, let me do it in the followup PR.